backend/app/adapter/sqldb/short_link_integration_test.go

```diff
@@ -47,6 +47,102 @@ type shortLinkTableRow struct {
 	twitterImageURL    *string
 }
 
+func TestShortLinkSql_UpdateTwitterTags(t *testing.T) {
+	title1 := "title1"
+	description1 := "description1"
+	imageURL1 := "url1"
+	title2 := "title2"
+	description2 := "description2"
+	imageURL2 := "url2"
+
+	testCases := []struct {
+		name              string
+		tableRows         []shortLinkTableRow
+		alias             string
+		metaTags          metatag.Twitter
+		expectedShortLink entity.ShortLink
+	}{
+		{
+			name: "OpenGraph tags not provided",
```

Comment:
It seems that the name suggest those are the tests for Open Graph tags.

Reply:
@byliuyang This is similar to what we did in `TestShortLinkSql_UpdateOpenGraphTags` test cases

Reply:
@kapillamba4 I didn’t see the value of this test case. Could you please elaborate?

Reply:
Resolved on Slack, will update the test cases